src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_params.py

```diff
@@ -32,6 +32,10 @@
 register_cli_argument('acr', 'location', location_type)
 register_cli_argument('acr', 'tags', tags_type)
 register_cli_argument('acr', 'admin_enabled',
+                      nargs='?',
+                      required=False,
+                      const='true',
+                      default=None,
                       help='Indicates whether the admin user is enabled',
                       choices=['true', 'false'])
```

Comment:
This is a formatting change that would apply to the whole file. If it's needed let's do it in another PR.